src/services/response-service.ts

```diff
@@ -12,7 +13,8 @@ import ResourceService from './resource-service'
 export default class ResponseService extends PercyClientService {
   resourceService: ResourceService
 
-  readonly ALLOWED_RESPONSE_STATUSES = [200, 201, 304]
+  readonly REDIRECT_STATUSES = [301, 302, 304, 307, 308]
+  readonly ALLOWED_RESPONSE_STATUSES = [200, 201, 301, 302, 304, 307, 308]
```

Comment:
Nice way to combine these? Is it even worth it?

Reply:
Instance initializers are run in the context of the constructor (at the very end to be specific), so you have access to `this`.

```
readonly ALLOWED_RESPONSE_STATUSES = [200, 201, ...this.REDIRECT_STATUSES]
```

Reply:
But I'm also not sure if these need to even be instance vars? If they don't change per instance and they aren't referenced by the instance outside of this file, couldn't these just be consts at the root level in this file?

Reply:
I agree, I’ll move it to a static const at the top